### ISSUE TITLE
failed to enumerate when trying to execute on Manjaro (Linux)

### ISSUE THREAD

**Issue by Gladox114** [GPSnoopy/RayTracingInVulkan#32]
After finally building it I can't execute it.
I cloned the git and installed glslang and libbacktrace-git to build it.
```
> ./RayTracer --benchmark --width 2560 --height 1440 --fullscreen --scene 1 --next-scenes --present-mode 0 
FATAL: failed to enumerate (ERROR_INITIALIZATION_FAILED)

 0# void Throw<std::runtime_error>(std::runtime_error const&) in ./RayTracer
 1# Vulkan::Check(VkResult, char const*) [clone .cold] in ./RayTracer
 2# Vulkan::Instance::GetVulkanPhysicalDevices() in ./RayTracer
 3# Vulkan::Instance::Instance(Vulkan::Window const&, std::vector<char const*, std::allocator<char const*> > const&, unsigned int) in ./RayTracer
 4# Vulkan::Application::Application(Vulkan::WindowConfig const&, VkPresentModeKHR, bool) in ./RayTracer
 5# Vulkan::RayTracing::Application::Application(Vulkan::WindowConfig const&, VkPresentModeKHR, bool) in ./RayTracer
 6# RayTracer::RayTracer(UserSettings const&, Vulkan::WindowConfig const&, VkPresentModeKHR) in ./RayTracer
 7# main in ./RayTracer
 8# __libc_start_main in /usr/lib/libc.so.6
 9# _start in ./RayTracer

```
I am having a gtx 1060 6gb with Nvidia-driver 460.32.03

**Comment by GPSnoopy** [comment]
Can you run any other Vulkan applications? It sounds like an incomplete Vulkan install.

**Comment by PMunkes** [comment]
Can you run vulkaninfo and post the results here? I think 460.32 doesn't support VK_KHR_raytracing_pipeline yet.

**Comment by Gladox114** [comment]
Seems like it was an Issue with some dependencies that need a reboot. Not sure if it was glslang or libbacktrace-git but I didn't reboot my PC and wanted to try it out. The build was successfull but after reboot It works now.

Really unnecessary this post, sorry guys.

**Comment by Gladox114** [comment]
> Can you run vulkaninfo and post the results here? I think 460.32 doesn't support VK_KHR_raytracing_pipeline yet, since that was only included from 460.89 on.
> [link](https://www.nvidia.com/en-us/geforce/news/quake-ii-rtx-vulkan-ray-tracing-game-ready-driver/)

```
> vulkaninfo | grep rayTracing
        rayTracingPipeline                                    = true
        rayTracingPipelineShaderGroupHandleCaptureReplay      = false
        rayTracingPipelineShaderGroupHandleCaptureReplayMixed = false
        rayTracingPipelineTraceRaysIndirect                   = true

```
And looking again at my Nvidia version I think something changed when installing a dependency. Now I have 460.39 for some reason.

**Comment by PMunkes** [comment]
The change could have been backported, since I don't think the changes were that big compared to the provisional version.
